### ISSUE TITLE
Warnings still thrown from java dev server

### ISSUE THREAD

**Issue by mattburns** [GoogleCloudPlatform/appengine-gcs-client#14]
Console output from the local dev server still has this warning:

> WARNING: The Google Cloud Storage Java API is deprecated and will soon be removed. Please use the Google Cloud Storage Client library instead. Migration documentation is available here: https://cloud.google.com/appengine/docs/java/googlecloudstorageclient/migrate

I think this is caused by `com.google.appengine.tools.cloudstorage.dev.LocalRawGcsService` still referencing `com.google.appengine.api.files.*`

This warning will mask any real dependencies I have in my code on the deprecated api...

**Comment by dt-rush** [comment]
Yep, this is definitely what's happening. It's unclear whether the devserver intends to keep this dependency on the local implementation of the files service. Would be great to get a comment of a project member on this, given the closely-impending deprecation.

**Comment by mattburns** [comment]
Agreed.
I've noticed the emails I'm getting from Google about the shutdown have the correct date for my last usage of the API. eg:

    Here’s the list of application(s) along with the most recent usage date of the API:
        s~oddprints, 2015-07-10 

So, if you think you've removed any code calling the API but your emails have a date after your fix, you better look again ;)

**Comment by thobrla** [comment]
The dev server dependency on the local Files API will be removed.
